src/core/_buttons.css

```diff
@@ -19,6 +19,11 @@ input[type="button"] {
   border-radius: var(--border-radius);
   border: var(--button-border);
   cursor: pointer;
+  -webkit-transition: border 0.2s;
```

Comment:
But I want to support webkit,moz,ms and o! How I can change this? In the build only webkit has been added.